client/functions.lua

```diff
@@ -63,54 +62,23 @@ function QBCore.Functions.DrawText3D(x, y, z, text)
     ClearDrawOrigin()
 end
 
-function QBCore.Functions.CreateBlip(id, data)
-    local blip = AddBlipForCoord(data.coords)
-
-    if data.sprite then SetBlipSprite(blip, data.sprite) end
-    if data.range then SetBlipAsShortRange(blip, data.range) else SetBlipAsShortRange(blip, true) end
-    if data.color then SetBlipColour(blip, data.color) end
-    if data.display then SetBlipDisplay(blip, data.display) end
-    if data.playername then SetBlipNameToPlayerName(blip, data.playername) end
-    if data.showcone then SetBlipShowCone(blip, data.showcone) end
-    if data.secondarycolor then SetBlipSecondaryColour(blip, data.secondarycolor) end
-    if data.friend then SetBlipFriend(blip, data.friend) end
-    if data.mission then SetBlipAsMissionCreatorBlip(blip, data.mission) end
-    if data.route then SetBlipRoute(blip, data.route) end
-    if data.friendly then SetBlipAsFriendly(blip, data.friendly) end
-    if data.routecolor then SetBlipRouteColour(blip, data.routecolor) end
-    if data.scale then SetBlipScale(blip, data.scale) else SetBlipScale(blip, 0.8) end
-
-    BeginTextCommandSetBlipName("STRING")
-    AddTextComponentString(data.name)
-    EndTextCommandSetBlipName(blip)
-
-    Blips[id] = {blip = blip, data = data}
-end
-
-function QBCore.Functions.RemoveBlip(id)
```

Comment:
But that change is not part of the PR, it may of been part of one of the commits as part of the PR. But if you look at the changes this PR will implement, the only changes it makes to master are the changes to the player.lua file. Obviously it needs properly rebasing before being merged, which I am happy to do if this change would be implemented if I implemented the locale system.

I can do this when I get home from my holiday/vacation in the next week 👍